library.java

```diff
@@ -23,11 +23,11 @@
 	private static final double maxFinesOwed = 1.0;
 	private static final double damageFee = 2.0;
 	
-	private static library SeLf;
-	private int BOOK_ID;
-	private int MEMBER_ID;
-	private int LOAN_ID;
-	private Date LOAN_DATE;
+	private static library self;// Changed the variable name to lowercase (camelback) by Sudeep Maharjan
+	private int bookId;// Changed the variable name to lowercase (camelback) by Sudeep Maharjan

```

Comment:
Please make changes to all the variables name
Eg change Book_ID to bookId in code below

Reply:
Sure

Reply:
Done